{{ cookiecutter.project_slug }}/docker-compose.yml

```
@@ -35,4 +34,4 @@ services:
 volumes:
   postgres:
   minio:
-  rabbitmq:
+  redis-data:
```

```suggestion
  redis-data:
```
Use a name consistent with the others.